index.html

```diff
@@ -1,33 +1,35 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title></title>
+	<title>lass</title>
 	<script src="https://kit.fontawesome.com/b8019b4db4.js" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
 	<header class="container">
-		<div id="logo">My Logo</div>
+		<div id="logo"><a href="#">My Logo</a></div>
 		<nav>
-			<button><i class="fas fa-bars" id="bar"></i></button>
+			<button type="button"><i class="fas fa-bars" id="bar"></i></button>
 		</nav>
 	</header>
 
 	<main class="container">
 		<section id="headline">
-			<h2>I’m Mariane
-				Glad to see you!</h2>
+			<h1>I’m Mariane
+				Glad to see you!</h1>
 
 			<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
 
 			<div id="connect">Let's Connect</div>
 			<div id="social">
-				<button><i class="fab fa-twitter"></i></button>
-				<button><i class="fab fa-linkedin-in"></i></button>
-				<button><i class="fab fa-github"></i></button>
-			</div>
-		</section>
+			<ul>
+			<li><a href="#"><i class="fab fa-twitter"></i></a></li> 
+			<li><a href="#"><i class="fab fa-github"></i></a></li> 
+			<li><a href="#"><i class="fab fa-facebook"></i></a></li> 
+			<li><a href="#"><i class="fab fa-reddit"></i></a></li> 
+			</ul></div>
```

Comment:
- [ ] OPTIONAL: I would also recommend increasing the icons size to match with Figma design: 

Your design  | Figma design
------------- | -------------
<img width="378" alt="Screenshot 2022-03-15 at 21 48 18" src="https://user-images.githubusercontent.com/77038610/158440006-90fb8f95-77d9-4a41-a410-0131121f3a68.png"> | <img width="378" alt="Screenshot 2022-03-15 at 21 48 40" src="https://user-images.githubusercontent.com/77038610/158440075-61cf1d99-81d5-4af2-9438-5993a6279bc6.png">